content/docs/atom-guide/look-dev/tools/material-canvas.md

```diff
@@ -0,0 +1,297 @@
+Input nodes represent named variables that will be added to the Material SRG structure with properties and connections exposed in the material type. These properties will be displayed and configurable in the material editor and material component. They can also be controlled through script. Input nodes have additional property slots for you to specify their name, description, and other data to make them easily identifiable. As with constant nodes, there are several different nodes corresponding to data types supported by AZSL and the material system.
+
+#### Vertex Nodes
+The vertex node category contains nodes that correspond to different vertex attributes like positions, normals, texture coordinates, and so on. There are nodes for the same attribute in local and world space.
+
+#### Function Nodes
+The function node category contains several math related and utility function nodes that process property and input values and return a result. Currently, the majority of the function nodes are wrappers for AZSL intrinsic functions.
+
+#### Texturing Nodes
+This category contains nodes related to texture sampling. Note that some texture sampling nodes sample from a constant vector by default. You may require input from a UV node or other varying vertex attribute node to sample using texture coordinates covering a surface or model.
+
+#### Scene Nodes
+This category is intended to contain nodes for elements of the Scene SRG. At this time, the only exposed node is Time.
+
+#### Utility Nodes
+The utility node category contains standard comment and group nodes used by all of the O3DE, graph editing tools. Comment nodes can be placed throughout the graph to leave notes and descriptions about a particular part of the graph. Group nodes can be used as a container for other nodes on the graph. Groups containing other nodes can then be expanded, collapsed, and treated as a single node.
+
+## Creating New Material Graph Nodes
+Material Canvas nodes, with the exception of utility nodes, are completely defined in JSON configuration files. As mentioned earlier, these files describe the nodes UUID, name, description, category, as well as the layout and details for each slot on the node. Node configurations may have additional settings or meta data to drive the code and data generation process.
+
+The material graph node document inspector supports creating unique node UUIDs, adding and removing slots, selecting data types, configuring default values, and managing custom settings for nodes and slots. The inspector also has custom controls for making selections and editing AZSL.
+
+It is possible to create nodes from scratch using tooling provided by Material Canvas. However, some of the nodes are simple enough that it might be easier and faster to copy an existing material graph node file, update the UUID, and make changes using the tool or directly in JSON. Future revisions to the material graph node inspector could further simplify this process.
```

Comment:
Remove sentence about future revisions, as those aren't set in stone.
```suggestion
It is possible to create nodes from scratch using tools provided by Material Canvas. However, some of the nodes are simple enough that it might be more convenient to copy an existing material graph node file, update the UUID, and make changes using the tool or directly in JSON.
```